### DISCUSSION TITLE
new update make the Project cant open

### DISCUSSION THREAD

**Discussion by TheMrBigWill** [TheHerobrine/DarkerNodes#72] [Q&A]
just me？
goes to 75% and Crash

`LoginId:75bf583e4fe7ae3b3f3f67ab2eb3f9aa
EpicAccountId:8dd9c1f5ae5d4faf8e5e6791b688d628

Assertion failed: FPaths::DirectoryExists(RealShaderDirectory) [File:D:/Build/++UE4/Sync/Engine/Source/Runtime/RenderCore/Private/ShaderCore.cpp] [Line: 1326] FPaths::DirectoryExists(../../../../../Project/A_TempFotTest/Plugins/DarkerNodes/Shaders)

UE4Editor_Core
UE4Editor_Core
UE4Editor_RenderCore
UE4Editor_RenderCore
UE4Editor_DarkerNodes!FDarkerNodesModule::StartupModule() [D:\Build\++Portal+Promotion\Sync\LocalBuilds\PluginTemp\HostProject\Plugins\DarkerNodes\Source\DarkerNodes\Private\DarkerNodes.cpp:14]
UE4Editor_Core
UE4Editor_Projects
UE4Editor_Projects
UE4Editor_Projects
UE4Editor
UE4Editor
UE4Editor
UE4Editor
UE4Editor
UE4Editor
kernel32
ntdll`

**Comment by hugoattal** [comment]
Shit, I knew it was a big update, but I expected it to be seemless... I'm investigating...

Really sorry for that, you can disable the plugin in your `.uproject` for the moment 😕...

**Comment by hugoattal** [comment]
I opened an issue for this: https://github.com/TheHerobrine/DarkerNodes/issues/73 I'll fix this by tomorrow

**Reply by TheMrBigWill** [reply]
glad to know. keep up
